inc/reception.class.php

```diff
@@ -83,7 +81,7 @@ public function getFromDBByOrder($plugin_order_orders_id)
         /** @var \DBmysql $DB */
         global $DB;
 
-        $query = "SELECT * FROM `" . $this->table . "`
+        $query = "SELECT * FROM `" . self::getTable() . "`
                WHERE `plugin_order_orders_id` = '" . $plugin_order_orders_id . "' ";
         if ($result = $DB->query($query)) {
```

Comment:
`query()` is normally deprecated, this type of format must be used (to be tested):
```suggestion
        $query = [
            'FROM' => self::getTable(),
            'WHERE' => ['plugin_order_orders_id' => $plugin_order_orders_id],
        ];
        if ($result = $DB->request($query)) {
```